### PR TITLE
add libmath dockerfile

### PR DESCRIPTION
**Pull request by ali4006**
Compiling `OpenLibm` libraries using verificarlo.

### CODE REVIEW

File: fuzzy/docker/level1/Dockerfile.libmath [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM verificarlo/verificarlo:latest
```
Comment (gkiar) [inline fuzzy/docker/level1/Dockerfile.libmath]: Can you instead build using the verificarlo container that we build here? I believe the name is `gkiar/fuzzy:ubuntu14.04-base`

File: fuzzy/docker/level1/Dockerfile.libmath [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+RUN apt-get update &&\
+    apt-get install -y make build-essential wget
```
Comment (gkiar) [inline fuzzy/docker/level1/Dockerfile.libmath]: I think `wget` and `build-essential` are in the container I mentioned using as a base, above. In which case, these could be removed.

**Comment by gkiar** [comment]
This is great, thanks @ali4006 !!

File: fuzzy/docker/level1/Dockerfile.libmath (hunk 2 above)
Comment (ali4006) [reply fuzzy/docker/level1/Dockerfile.libmath]: Yes, I changed docker base.

**Comment by gkiar** [comment]
@ali4006  - I accidentally merged before commenting with this, but could you please add a section to the `.travis.yml` so that this image gets built, and open a new PR?

**Pull request merged**